docs/protocol-spec.md

```diff
@@ -5,17 +5,16 @@ Bindle uses HTTP/2 with TLS as a transport protocol. All bodies and responses ex
 The HTTP endpoints defined above MAY exist as a subpath on a server, or in the server's root. For example, `https://example.com/v1/_i/foo` and `https://example.com/_i/foo` are both legal paths for the specification below. However, `https://example.com/_i/v1/foo` is not (or, rather, it is a legal URI for a package named `v1/foo`).
 
 HTTP Endpoints:
-- `/_i/{bindle-name}`: The path to a bindle's invoice. Note that {bindle-name} can be pathy. For example, `/_i/example.com/mybindle/1.2.3` is a valid path to a bindle named `example.com/mybindle/1.2.3`.
+- `/_i/{bindle-name}`: The path to a bindle's invoice. Note that `{bindle-name}` can be pathy. For example, `/_i/example.com/mybindle/1.2.3` is a valid path to a bindle named `example.com/mybindle/1.2.3`.
     - `GET`: Get a bindle by name. This returns an invoice object.
     - `HEAD`: Send just the headers of a GET request
     - `DELETE`: Yank a bindle. This will set the `yank` field on a bindle to `true`. This is the only mutation allowed on a Bindle.
 - `/_i`
     - `POST`: Create a new bindle, optionally also sending some or all of the parcels. If all of the parcels specified in the bindle exist, a 201 status will be returned. If 1 or more of the parcels are missing, a 202 status will be returned with a reference to the missing parcels
-- `/_p/{parcel-id}`: The path to a parcel ID, where `{parcel-id}` is an exact SHA to a parcel.
+- `/_i/{bindle-name}@{parcel-id}`: The path to a Bindle name and parcel ID, where `{parcel-id}` is an exact SHA of a parcel and `{bindle-name}` follows the same rules as outlined above. Parcels can only be accessed if the client has the proper permissions to access the given bindle and, as such, cannot be accessed directly
     - `GET`: Directly fetch a parcel's opaque data.
     - `HEAD`: Send just the headers of a GET request
-- `/_p`
-    - `POST`: Create a parcel if it does not already exist. This may be disallowed. This MUST post both the label and the parcel data with a `multipart/form-data` content type, with the label data as the first part in the request
+    - `POST`: Create a parcel if it does not already exist. This may be disallowed. The data included in the body must have the same SHA as indicated by the `{parcel-id}` and must exist within the invoice
```

Comment:
I'm feeling like this part of the spec is coming together nicely. Seeing this in writing, I think we made the right choice, and I am glad we got it done this early